package.json

```diff
@@ -117,15 +117,148 @@
                     "description": "How much to extend neovim viewport on each side beyond vscode editor size. Increase if you are not seeing decorations or highlights on the edges of the editor."
                 },
                 "vscode-neovim.useCtrlKeysForNormalMode": {
+                    "deprecationMessage": "Use vscode-neovim.ctrlKeysForNormalMode instead (Set it to empty)",
                     "type": "boolean",
                     "default": true,
                     "description": "Use Ctrl keys in normal/visual/etc...(except insert) modes"
                 },
                 "vscode-neovim.useCtrlKeysForInsertMode": {
+                    "deprecationMessage": "Use vscode-neovim.ctrlKeysForInsertMode instead (Set it to empty)",
                     "type": "boolean",
                     "default": true,
                     "description": "Use Ctrl keys in insert mode"
                 },
+                "vscode-neovim.ctrlKeysForNormalMode": {
+                    "type": "array",
+                    "uniqueItems": true,
+                    "items": {
+                        "type": "string",
+                        "enum": [
+                            "a",
+                            "b",
+                            "c",
+                            "d",
+                            "e",
+                            "f",
+                            "g",
+                            "h",
+                            "i",
+                            "j",
+                            "k",
+                            "l",
+                            "m",
+                            "n",
+                            "o",
+                            "p",
+                            "q",
+                            "r",
+                            "s",
+                            "t",
+                            "u",
+                            "v",
+                            "w",
+                            "x",
+                            "y",
+                            "z",
+                            "/",
+                            "]",
+                            "right",
+                            "left",
+                            "up",
+                            "down",
+                            "backspace",
+                            "delete"
+                        ]
+                    },
+                    "default": [
```

Comment:
I think we want to keep delete, left/right, etc.

Reply:
This is not enabled by default and users can enable it themselves. Need to delete?

Reply:
Because this configuration is mainly to help users enable shortcuts more easily and avoid doing too much configuration in keybindings.json, it is best to keep these keys.